control-catalog/storage/object/controls.md

```diff
@@ -0,0 +1,5 @@
+| Control Id | Objective | Description | Test | Service Taxonomy Id | NIST CF | MITRE ATT&CK Mitigations | Threats |
```

Comment:
This is an interesting question. On `Detect`, I can see benefit in assessing the CSP's native threat detection capabilities (as well as basic audit logging of specific APIs) for a given service - e.g. data stores exposed outside of given data perimeters - here we'd also have the option of mapping to MITRE ATT&CK `Data Sources` e.g. for Cloud Storage: https://attack.mitre.org/datasources/DS0010/  

However...this would naturally be a lot more work...It feels like our priority should be on maturing our thinking on `Protect` with more `Protect` examples within and across the services and continuing to think/experiment on `Detect`? Feels like an early decision for steerco too.

Reply:
Totally agree that the `Test ` values in Gherkin don't read well in that format. We can shift to a separate gherkin file as you suggest.